### DISCUSSION TITLE
Release branches nowhere to be seen!

### DISCUSSION THREAD

**Discussion by moshmodo** [actions/virtual-environments#5978] [General questions] [answered]
I'm wondering why all of your branches have been missing since today (2022-07-27)? 

Are you changing the way you manage them?

I did not find any ongoing discussions regarding that.

Thanks!

**Comment by ddobranic** [comment, marked as answer]
Hello @simbo1984 , all the release branches have been restored. Sorry for the inconvenience.

**Reply by miketimofeev** [reply]
@simbo1984 could you please clarify your usage scenario with branches? Our intention is to keep only the main branch.

**Reply by moshmodo** [reply]
@miketimofeev Back then, you committed your changes in the master branch as you go along while in the branches it's what you deploy for real. We had issues a couple of times where the master was not OK while the release branch was.

And in the branches there could be some diffs by type (windows/linux) and sometimes quick fixes without repeating everything that was committed in the master branch.

As I recall we followed your official doc at that time.

**Reply by miketimofeev** [reply]
We recommend to use the latest main branch in all the cases as it contains bug fixes that could exist in the release branches

**Reply by moshmodo** [reply]
The release branches are more stable than the master branch according to what we have experienced many times. Yes the master contains all the fixes (like the weekly branches) but also sometimes other new commits which sometimes can create other problems. So the stability is more random than the branches (based on our experience).

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
